pkg/kine/drivers/generic/admission.go

```diff
@@ -18,16 +18,27 @@ type AdmissionControlPolicy interface {
 }
 
 const (
-	statusAccepted string = "accepted"
-	statusDenied   string = "denied"
+	// operation was evaluated by the admission control and accepted
+	resultAccepted string = "accepted"
+	// operation was evaluated by the admission control and denied by policy
+	resultDenied string = "denied"
 )
 
+// Queries that perform write operations on the database
+var writeQueries = map[string]struct{}{
```

Comment:
Having a second look at it, this could also be a `map[string]bool`. That way we can use the fact that looking up a missing key will return a zero value.

Reply:
Done